tests/Microsoft.CodeAnalysis.Testing/Directory.Build.props

```diff
@@ -21,7 +21,7 @@
   <Import Project="$([MSBuild]::GetPathOfFileAbove('Directory.Build.props', '$(MSBuildThisFileDirectory)../'))" />
 
   <PropertyGroup>
-    <TestTargetFrameworks>netcoreapp3.1;net472;net46;netcoreapp1.1;netcoreapp2.0</TestTargetFrameworks>
+    <TestTargetFrameworks>netcoreapp3.1;net472</TestTargetFrameworks>
```

Comment:
📝 This part should be reverted. The testing library still supports old versions of Roslyn.